packages/react-core/src/components/Wizard/WizardStep.tsx

```diff
@@ -16,8 +16,8 @@ export interface WizardStepProps {
   id: string | number;
   /** Optional for when the step is used as a parent to sub-steps */
   children?: React.ReactNode | undefined;
-  /** Props for WizardBody that wraps content by default. Can be set to null for exclusion of WizardBody. */
-  body?: Omit<Omit<WizardBodyProps, 'children'>, 'children'> | null;
+  /** Props for WizardBody that wraps content by default. */
+  body?: Omit<Omit<WizardBodyProps, 'children'>, 'children'>;
```

Comment:
Should the prop also be marked as required so that `undefined` can't be passed / the prop omitted? It sounds that way based on the issue, but I could be misunderstanding.

Reply:
I think it being optional should be fine. I'll try locally, but the logic in WizardToggle checks whether `body || body === undefined` to render the WizardBody wrapping `children`. We could probably just remove that conditional now,, though, since we should always want the WizardBody wrapper.

Reply:
Just to clarify, the `body` prop in WizardStep is for passing an object of WizardBody props in; the prop name isn't exactly descriptive of the actual behavior. May be worth updating the prop to `bodyProps` instead at some point.